app/src/main/java/io/apicurio/registry/rest/v2/DownloadsResourceImpl.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.rest.v2;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.interceptor.Interceptors;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import io.apicurio.registry.auth.Authorized;
+import io.apicurio.registry.auth.AuthorizedLevel;
+import io.apicurio.registry.auth.AuthorizedStyle;
+import io.apicurio.registry.logging.Logged;
+import io.apicurio.registry.metrics.health.liveness.ResponseErrorLivenessCheck;
+import io.apicurio.registry.metrics.health.readiness.ResponseTimeoutReadinessCheck;
+import io.apicurio.registry.rest.v2.shared.DataExporter;
+import io.apicurio.registry.storage.DownloadNotFoundException;
+import io.apicurio.registry.storage.RegistryStorage;
+import io.apicurio.registry.storage.dto.DownloadContextDto;
+import io.apicurio.registry.storage.dto.DownloadContextType;
+import io.apicurio.registry.types.Current;
+
+/**
+ * @author eric.wittmann@gmail.com
+ */
+@ApplicationScoped
+@Interceptors({ResponseErrorLivenessCheck.class, ResponseTimeoutReadinessCheck.class})
+@Logged
+@Path("/apis/registry/v2/downloads")
+public class DownloadsResourceImpl {
```

Comment:
Note:  I haven't added `/downloads` to the openapi spec.  I thought maybe it should not be included.  But now after thinking about it more, I'm not sure.  @Apicurio/developers what do you think, should this endpoint be in the openapi?